dashboard/src/components/TestsTable/TestsTable.tsx

```diff
@@ -255,14 +258,17 @@ export function TestsTable({
     [filterCount, intl, filter],
   );
 
-  // TODO: there should be a filtering for the frontend before the backend AND that filtering should consider the individual tests inside each test "batch" (the data from individualTestsTables), not only the rows of the external table
+  // TODO: there should be a filtering for the frontend before the backend
+  // that frontend filtering should consider the individual tests inside each test batch (the data from individualTestsTables),
+  // not only the rows of the external table (same for BootsTable)
```

Comment:
We are not using this approach anymore, we will just preserve the data, add a spinner until the fresh data is retrieved